velox/common/memory/tests/MemoryManagerTest.cpp

```diff
@@ -92,10 +92,10 @@ TEST_F(MemoryManagerTest, Ctor) {
 namespace {
 class FakeTestArbitrator : public MemoryArbitrator {
  public:
-  FakeTestArbitrator(const Config& config, int64_t capacity)
+  FakeTestArbitrator(const Config& config)
       : MemoryArbitrator(
             {.kind = config.kind,
-             .capacity = capacity,
+             .capacity = config.capacity,
```

Comment:
Let's keep FakeTestArbitrator and set a larger capacity to verify the check in memory manager (<=)? Thanks!

Reply:
The default value is large. But I'll make it explicit so that it be more illustrative in the test.